### PR TITLE
Fix confusing u-url, and add explicit syndication links

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Description

- simplified author `h-card` to avoid `u-url` confusion
- added hidden `u-url` and optional `u-syndication` links to pages
- added syndication links to several recent articles

## Steps to test/reproduce

- use https://indiewebify.me/validate-h-entry/ to test the h-entry data from recent posts
- if you test on the live site, you'll see it treats the author url as the post url, which is the main bug being fixed here

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[author u-url is overriding post url, breaking webmentions on a post](https://trello.com/c/ZK2AJT2r/168-author-u-url-is-overriding-post-url-breaking-webmentions-on-a-post)

File: content/_includes/page/base.njk [hunk 1]
```diff
@@ -24,4 +24,10 @@
     {{ content | typogr | safe }}
     {% include "page/list.njk" %}
   </div>
+
+  <!-- used for webmentions -->
+  <a href="{{ page.url | url }}" class="u-url" hidden>{{ full_title | safe }}</a>
+  {% for link in syndication %}
+    <a rel="syndication" class="u-syndication" href="{{ link | url }}" hidden>{{ link | url }}</a>
+  {% endfor %}
```
Comment (stacyk) [inline content/_includes/page/base.njk:32]: @mirisuzanne Is the first <a> on the page after the ` <!-- used for webmentions -->` here missing the title? This screenshot is from the deploy preview on Ed's article:  https://deploy-preview-218--oddleventy.netlify.app/2022/06/01/dependabot-single-pull-request/

Screen Shot 2022-06-30 at 3 21 27 PM

File: content/blog/2022/dependabot-single-pull-request.md [hunk 2]
```diff
@@ -47,11 +49,11 @@ to upgrade dependencies at our convenience.
...
-1. Configure the runtimes required by our project (Python and Node)
-1. Run a custom command to upgrade all dependencies
-1. Use git to detect changed files
-1. Commit the changes to a pre-defined branch (if they exist)
-1. Open a pull request for that branch (if it doesn't exist already)
+2. Configure the runtimes required by our project (Python and Node)
+3. Run a custom command to upgrade all dependencies
+4. Use git to detect changed files
+5. Commit the changes to a pre-defined branch (if they exist)
+6. Open a pull request for that branch (if it doesn't exist already)
```
Comment (jgerigmeyer) [inline content/blog/2022/dependabot-single-pull-request.md:56]: Was this change on purpose? This was an intentional use of Markdown's auto-numbering: https://daringfireball.net/projects/markdown/syntax#list

Comment (mirisuzanne) [reply content/blog/2022/dependabot-single-pull-request.md:56]: nope, this must have been a linter?

**Review by jgerigmeyer — APPROVED** [review]
